src/init/features/dataconnect/sdk.ts

```diff
@@ -101,7 +104,9 @@ export async function doSetup(setup: Setup, config: Config): Promise<void> {
     const outputDir = await promptOnce({
       message: `What directory do you want to write your Kotlin SDK code to? (If not absolute, path will be relative to '${connectorInfo.directory}')`,
       type: "input",
-      default: newConnectorYaml.generate.kotlinSdk?.outputDir,
+      default:
+        newConnectorYaml.generate.kotlinSdk?.outputDir ||
+        `./../.dataconnect/generated/${newConnectorYaml.connectorId}/kotlin-sdk`,
```

Comment:
For Kotlin SDK, it would be better to use `./../.dataconnect/generated/${newConnectorYaml.connectorId}/kotlin-sdk/src/main/kotlin/${newConnectorYaml.connectorId}`

Reply:
Sure~